Formula/c/cspell.rb

```diff
@@ -0,0 +1,20 @@
+class Cspell < Formula
+  desc "Spell checker for code"
+  homepage "https://cspell.org"
+  url "https://registry.npmjs.org/cspell/-/cspell-8.17.4.tgz"
+  sha256 "706660433488a9889c2b81e1a10c86a0595b742ccd4815077af749f3aaa70d61"
+  license "MIT"
+
+  depends_on "node"
+
+  def install
+    system "npm", "install", *std_npm_args
+    bin.install_symlink Dir["#{libexec}/bin/cspell"]
```

Comment:
I think we'd prefer wildcard, cc @branchvincent 

```suggestion
    bin.install_symlink Dir["#{libexec}/bin/*"]
```

Reply:
I changed this. Because otherwise it will also install a binary called `cspell-esm`, which is identical to cspell.

Reply:
Does it hurt anything to keep it? Ideally we'd install the same things that `npm i -g cspell` would. Upstream may have a good reason to install it, and if they don't maybe they would accept a PR to remove it: https://github.com/streetsidesoftware/cspell/blob/6a58c37567ba2419db73e790979eba61721b1832/packages/cspell/package.json#L6-L9

Reply:
The duplicated binaries are due to backward compatibility reason. `cspell` and `cspell-esm` used to have different implementation (common js vs esm js).  Since this is a new formula in homebrew, I don't think we need to expose the cspell-esm binary, which no one uses.